.github/workflows/main.yml

```diff
@@ -1,4 +1,4 @@
-on: [push]
+on: [push, pull_request]
```

Comment:
I'm sneaking this change into your PR @Alexander-N, since without it it seems our CI won't run your contribution.

This is also a valuable contribution for you, because you're the first outside contributor to help us since we switched from travis to github actions :)